go/enclave/rollupchain/rollup_chain.go

```diff
@@ -767,6 +745,41 @@ func (rc *RollupChain) IsAccountContractAtBlock(accountAddr gethcommon.Address,
 	return len(chainState.GetCode(accountAddr)) > 0, nil
 }
 
+func (rc *RollupChain) ExecuteOffChainTransactionAtBlock(apiArgs *gethapi.TransactionArgs, blockNumber gethrpc.BlockNumber) (*core.ExecutionResult, error) {
+	// TODO review this during gas mechanics implementation
+	callMsg, err := apiArgs.ToMessage(rc.GlobalGasCap, rc.BaseFee)
+	if err != nil {
+		return nil, fmt.Errorf("unable to convert TransactionArgs to Message - %w", err)
+	}
+
+	hs := rc.storage.FetchHeadState()
+	if hs == nil {
+		panic("Not initialised")
+	}
+	// todo - get the parent
+	r, f := rc.storage.FetchRollup(hs.HeadRollup)
+	if !f {
+		panic("not found")
```

Comment:
I know it's only testnet but does this add a way to easily kill the node from a RPC method? (Try to execute call against a future rollup num)

Reply:
I'm fairly certain that this was here before. That being said, I totally agree, it's a good time to fix this :)

Reply:
ah balls haha, yeah you're right sorry. But yeah good to fix it at some point if not now